fighthealthinsurance/medicaid_api.py

```
@@ -0,0 +1,931 @@
+            missing.append(
+                "How many years did you (or spouse or ex-spouse) work and pay medicare taxes?"
+            )
+        elif years_worked > 10:
+            eligible_medicare = True
+        elif on_medicaid_past is None:
+            missing.append("Were you on medicaid previously?")
+        elif on_medicaid_past or esrd or als or receiving_ssdi:
+            eligible_medicare = True
+        else:
+            eligible_medicare = False
+            alternatives.append(
```

Ruff (0.12.2)

Undefined name `alternatives`

(F821)